packages/@coorpacademy-components/src/molecule/bulk-progress-bar/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ProgressBar from '../progress-bar';
+import StatusItem from '../../atom/status-item';
+import style from './style.css';
+import {Props, propTypes} from './types';
+
+const STATUS = {
+  inProgress: 'inProgress',
+  fail: 'fail'
+};
+
+const renderStatusIcon = (status: string, progress: number) => {
+  if (status === STATUS.inProgress)
+    return <div className={style.progressText}>{`${progress} %`}</div>;
+  return <StatusItem icon={status === STATUS.fail ? 'invalid' : 'valid'} />;
+};
+
+const resolveProgressBarColor = (status: string) => {
+  if (status === STATUS.fail) return '#ED3436';
+  else if (status === STATUS.inProgress) return '#0061FF';
+  else return '#35CC7F';
```

Comment:
tu peux mettre ces couleurs dans des constantes tout en haut en leur donnant le meme nom de variables qui existe dans le fichier colors.css, ca aura plus de sens je pense :) 
packages/@coorpacademy-components/src/variables/colors.css 
pa exemple : 
```suggestion
  const cm_positive_100 = '#35CC7F'
```